peopleInsights/examples/PeopleInsightsZone.json

```diff
@@ -13,6 +13,9 @@
              "dwellTime": 45
            }
          ],
+          "objectsCount": {
```

Comment:
Curiosity number of object calculation entails that:
- we need to update all P100/Curiosity to the latest version before you can use data discovery. Whenever we introduce a new property.
- the information is redundant and can be computed. This increases slightly the payload = data traffic.

With an Azure function (or similar) we would support older Curiosity versions as well. What speaks against that option?

Reply:
@florian-fischer-swarm @mecraggs thoughts?